Update tiny11maker.ps1 (Param fix, unique log file with date)

deftoner opened the pull request:
Fixed the Define parameters, moved to the top (if not will print error, param needs to be set at the beginning) 

Also changed the log output from tiny11.log adding date and time, for better troubleshoot, and to avoid file lock or exist from a previous attempt to run the tool.

Karl-WE commented:
@deftoner appreciate the changes and confirm they are good.
Why replacing Start-Transscript? Without modifying the script there seems to be no logging at all.
What about editing your PR adding a new param?

deftoner commented:
> @deftoner appreciate the changes and confirm they are good. Why replacing Start-Transscript? Without modifying the script there seems to be no logging at all. What about editing your PR adding a new param?

Hello! thanks for looking into this. 

Looks like I didnt save the laste version, didnt want to replace the Start-Transscript, I actually moved it after naming the file with date and time, but looks that I made a mistake and the latest version of the file wasnt saved. 

I fixed that, added the missing command, and also added the variable for the release date at the top for easy update on new releases. 

Hope this helps

Karl-WE commented:
Will Review, thanks!
